docs/content/installation/install_compatibility_requirements.md

```diff
@@ -49,9 +49,9 @@ Complete these steps to prepare your environment for installing the CSI (Contain
 
    `oc apply -f 99-ibm-attach.yaml`
 
-2. Configure your storage system host attachment, per worker node.
+2. (Optional): Configure your storage system host attachment, per worker node.
 
-    **Important:** The CSI driver does not define hosts on your storage system.
+    **Note:** As of IBM® block storage CSI driver 1.10 release, host attachment is now automatic on new host connectivity, if the HostDefinition custom resource is installed. For more information and installation instructions, see [Installing the `HostDefinition` custom resource](install_hostdefinition.md). If this CR is not installed, the CSI driver does not define hosts on your storage system.
```

Comment:
(volume to) host `attachment` has always "automatic" (dynamic), this feature is strictly about host `definition`, no changes to the attachment (/publish/map) feature

Reply:
updated

Reply:
I still see `attachment` there (`defining the host attachment is now automatic for new host connectivity`).
I don't think `attachment` is relevant to this feature (and I don't think `connectivity` is relevant in this note either)

Reply:
@oriyarde  @matancarmeli7 
the original note for this prerequisite was: The CSI driver does not define hosts on your storage system.
is this something that now needs to be changed due to host definition feature?
Please see [Step 2](https://www.ibm.com/docs/en/stg-block-csi-driver/1.9.0?topic=installing-compatibility-requirements) and let me know if this needs to be updated or not (and how)

Reply:
IMO we shouldn't have used the term "attachment" for host definition, even before this feature

Reply:
see my explanation [here](https://github.com/IBM/ibm-block-csi-driver/pull/517#pullrequestreview-1024601964)
HIH

Reply:
LGTM